scripts/container-only/.wkdev-init

```diff
@@ -206,6 +206,16 @@ try_setup_permissions_jhbuild_directory() {
 
 }
 
+try_setup_permissions_rust_directory() {
+
+    echo ""
+
+    local rust_directory="/opt/rust"
+    echo "[12/13] Setup rust '${rust_directory}' directory permissions..."
```

Comment:
13 is usually after 12 :).

You have to bump versions manually. Or make a new function so this isn't manual every time.